db/migrate/.rubocop.yml

```diff
@@ -2,3 +2,7 @@ inherit_from: ../../.rubocop.yml
 
 Naming/VariableNumber:
   CheckSymbols: false
+
+Rails/ThreeStateBooleanColumn:
```

Comment:
This is working around the same issue mentioned in https://github.com/mastodon/mastodon/pull/30836

The presence of this config file is unintentionally disabling this cop here, due to how `Include` merges in the settings. For now, I've added this explicit opt-in ... better long term idea is probably to delete this file and move the `Naming` config back to root config instead of this dir only.